core/storage/exploration/gae_models.py

```diff
@@ -372,8 +464,17 @@ def get_non_private(cls):
 
     @classmethod
     def get_top_rated(cls, limit):
-        """Returns an iterable with the top rated exp summaries that are
+        """Fetches the top rated exp summaries that are
         public in descending order.
```

Comment:
Move some of this to the previous line (which looks a bit short).

Also, in descending order of what?